tox.ini

```diff
@@ -34,7 +34,9 @@ commands= py.test --genscript=pytest1
 [testenv:flakes]
 basepython = python2.7
 deps = flake8
+     restructuredtext_lint
```

Comment:
OK, we will just have to change the docs as well. TBH I don't think it matters much, but if you think it is worthwhile I will gladly rename it.